resource-manager/src/main/java/org/ihtsdo/otf/resourcemanager/ResourceManager.java

```diff
@@ -107,6 +108,21 @@ public InputStream readResourceStream(final String resourcePath) throws IOExcept
 		}
 	}
 
+	public File doReadResourceFile(String resourcePath) {
+		try {
+			InputStream inputStream = this.readResourceStream(resourcePath);
```

Comment:
@pgwilliams I created several "doX" methods to return null instead of throwing an exception. These have been added to ResourceManager.